src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -187,10 +189,12 @@
 short-summary: Remove a network rule.
 examples:
   - name: Remove a network rule. (autogenerated)
-    text: az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet mysubnet
+    text: |
+        az storage account network-rule remove --account-name MyAccount --resource-group MyResourceGroup --subnet MySubnetID
     crafted: true
   - name: Remove a network rule. (autogenerated)
-    text: az storage account network-rule remove --account-name MyAccount --ip-address 23.45.1.0/24 --resource-group MyResourceGroup
+    text: |
+        az storage account network-rule remove --account-name MyAccount --ip-address 23.45.1.0/24 --resource-group MyResourceGroup
```

Comment:
can we give more descriptive example name?

Reply:
It's an Aladdin planning feature. For now, we have to accept the name it gave or reject this example.

Reply:
This time I can accept it.